mmseg/models/backbones/bisenet.py

```diff
@@ -0,0 +1,677 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import warnings
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, build_activation_layer, build_norm_layer
+from mmcv.runner import BaseModule
+
+from mmseg.ops import resize
+from ..builder import BACKBONES
+
+
+class DetailBranch(BaseModule):
+    """Detail Branch with wide channels and shallow layers to capture low-level
+    details and generate high-resolution feature representation.
+
+    Args:
+        detail_channels (Tuple[int]): Size of channel numbers of Stage 1,
+            Stage 2 and Stage 3 in Detail Branch.
+            Default: (64, 64, 128)
+        in_channel (int): Channel of input image. Default: 3
+        conv_cfg (dict | None): Config of conv layers.
+            Default: None
+        norm_cfg (dict | None): Config of norm layers.
+            Default: dict(type='BN')
+        act_cfg (dict): Config of activation layers.
+            Default: dict(type='ReLU')
+
+    Returns:
+        feat (Tensor): Feature map of Detail Branch.
+    """
+
+    def __init__(self,
+                 detail_channels,
+                 in_channel=3,
+                 conv_cfg=None,
+                 norm_cfg=dict(type='BN'),
+                 act_cfg=dict(type='ReLU'),
+                 init_cfg=None):
+        super(DetailBranch, self).__init__(init_cfg)
+        detail_branch = []
+        for i in range(len(detail_channels)):
+            if i == 0:
+                detail_branch.append(
+                    nn.Sequential(
+                        ConvModule(
+                            in_channels=in_channel,
+                            out_channels=detail_channels[i],
+                            kernel_size=3,
+                            stride=2,
+                            padding=1,
+                            bias=False,
```

Comment:
Maybe we do not need to add bias explicit since it will be handled by ConvModule.